<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectQueryUI</class>
 <widget class="QWidget" name="SelectQueryUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabele valabile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabele selectate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloane valabile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloane selectate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonAddTab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adauga Tabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonRemTab">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Scoate Tabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonAddCol">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adauga Coloana</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonRemCol">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Scoate Coloana</string>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListTabVal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListTabSel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListColVal">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>151</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListColSel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>151</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonExe">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
